tests/syntax_error_tests.rs

```diff
@@ -48,9 +48,4 @@ fn test_syntax_errors() {
     test_1_error("${3}", "Quantifier not allowed here");
     test_1_error("(?=abc)*", "Quantifier not allowed here");
     test_1_error("(?!abc){3,}", "Quantifier not allowed here");
-
-    test_1_error(
```

Comment:
Let's restore this; I don't think we should support the non-Unicode paths where escapes are conditionally backreferences or decimal.

Reply:
I restored the test. The error message is different but the result is the same.